dev/tools/localization/bin/gen_l10n.dart

```diff
@@ -23,10 +23,14 @@ void main(List<String> arguments) {
   parser.addOption(
     'arb-dir',
     defaultsTo: path.join('lib', 'l10n'),
-    help: 'The directory where all localization files should reside. For '
-      'example, the template and translated arb files should be located here. '
-      'Also, the generated output messages Dart files for each locale and the '
-      'generated localizations classes will be created here.',
+    help: 'The directory where the template and translated arb files should '
+      'be located.',
+  );
+  parser.addOption(
+    'output-dir',
+    defaultsTo: path.join('lib', 'l10n'),
+    help: 'The directory where the generated output messages Dart files for '
```

Comment:
Maybe a little more prescriptive: The directory where the generated localization classes will be written. The app must import the [how to refer to the app_localizations.dart file?] from this directory.